src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.scss

```diff
@@ -3,3 +3,22 @@
 .AnnotationThreadContent-loading {
     padding: $bdl-grid-unit * 5 0;
 }
+
+.AnnotationThread {
+    .bcs-ActivityThread {
+        position: relative;
+        max-height: $bdl-grid-unit * 132;
```

Comment:
Should we share this value with the other activity types? How are we keeping the threads and reply forms consistent?

Reply:
This value sets the maximum height of the Activity Thread only when used in `AnnotationThread`. The reasoning behind it is that when Annotation is displayed in a "bubble" over the Canvas Board we don't want the thread to take too much vertical space. Similar limitation is not needed in other activity types that are displayed in Sidebar.
I've also changed the value to a static one.